workbench/src/renderer/components/DataDownloadModal/index.jsx

```diff
@@ -152,6 +153,7 @@ export class DataDownloadModal extends React.Component {
       selectedLinksArray,
       dataRegistry,
     } = this.state;
+    const { t } = this.props;
```

Comment:
I was expecting to see `t` passed as a prop to this component in the `App`'s render method. Otherwise `t` will be `undefined` here right?

Reply:
Ahhh I get it, the parent component is now `withTranslation()`, which must be passing `t` to it's child.

Reply:
Exactly!